src/states/node.rs

```diff
@@ -550,6 +554,34 @@ impl Node {
         self.handle_signed_message(content, route, hop_name, &sent_to)
     }
 
+    fn ack_and_broadcast(&mut self,
```

Comment:
Doc: acknowledge reception from another group, and forward to another group or a node within our own group.